CONTRIBUTING.adoc

```diff
@@ -3,128 +3,52 @@ Contributing to the Globus SDK
 
 First off, thank you so much for taking the time to contribute! :+1:
 
-This document is a set of guidelines for making modifications to the SDK, not a
-set of strict rules.
-Sometimes it's okay to go off-book if you have a good reason, so use your best
-judgement.
-Feel free to propose changes to this document in a pull request.
+Reporting Bugs & Requesting Features
+------------------------------------
 
-QuickStart
-----------
-
-Impatient? Want to get hacking on this right away?
-A few quick rules if you don't want to read this full document:
-
-  - All of your code *must* pass `flake8`. We won't consider merging code which
-      doesn't pass `flake8`
   - Check if there's a matching
       https://github.com/globus/globus-sdk-python/issues[issue]
       before opening a new issue or pull request
-  - Any features which change the public API of the SDK must include
-      documentation updates
-  - `make test` must pass
-
-Reporting Bugs
---------------
-
-We welcome any and all bugs on the
-https://github.com/globus/globus-sdk-python/issues[GitHub Issue Tracker].
-
-However, the best bug reports are ones which follow the following rules:
-
-  - *Check for duplicates*. You don't have to scour our issue tracker, but do a
-      quick search to try to make sure you aren't reporting a known bug.
-  - *Use a clear descriptive title* for the issue you are reporting. The best
-      titles are short, but also uniquely describe the problem.
-  - *Provide a specific example of how to reproduce*. Example code is best, but
-      even a vague description is better than no description.
-  - *Explain what behavior you expected* and why. Not everything unexpected is
-      a bug -- what appears to you as a defect may in fact be faulty
-      documentation, for example, which needs to be updated.
-  - *Include a stacktrace* where applicable. If you're getting an error from
-      within the SDK code itself, stacktraces often help us figure out what's
-      going on much faster.
-
-You can also help us a lot by including the following information:
+  - When possible, provide a code sample to reproduce bugs
 
-  - *What version of python are you running?* The SDK supports a wide variety
-      of python versions, and a wide class of bugs are results of version
-      incompatibilities between those versions.
-  - *What OS and OS version are you running?* `uname -a` is the easiest way to
-      give us this info, but additional description like "Ubuntu Linux 15.10"
-      may help.
-  - *What other python packages do you have installed?* The best thing in this
-      case is to show us the results of `pip freeze`
+Linting & Testing
+-----------------
 
+- All code is autoformatted with [`black`](https://github.com/ambv/black) and
+    [`isort`](https://github.com/timothycrosley/isort). You may run
```

Comment:
```suggestion
   https://github.com/timothycrosley/isort[isort]. You may run
```

Ignore these if you did this on purpose but it looks like they just might be accidental markdown links instead of adoc.